source/_integrations/ps4.markdown

```diff
@@ -228,3 +217,10 @@ Be sure to include the following information:
 As well as the exact values for the following attributes found in the state of your PS4 entity.
 - media_title
 - media_content_id
+
+## Advanced Use
+
+### Ports
+This integration uses UDP port 1987 as the source port during runtime. During configuration, the source port will be UDP port 1988. These port assignments can be used to configure firewall rules. They can also be useful for mapping ports in Docker when not using the option `--net=host`.
+
+In the event that these ports cannot be used, the ports will fallback to a random port.
```

Comment:
Adding a new line. Furthermore, removed the Docker networking running mode line, as Home Assistant only supports running on the host network and no other modes.

```suggestion

This integration uses UDP port 1987 as the source port during runtime. During configuration, the source port will be UDP port 1988. These port assignments can be used to configure firewall rules.

In the event that these ports cannot be used, the ports will fallback to a random port.
```

Reply:
Thanks frenck. Sounds good to me. Commited your suggestions.